docs/userguide/howto.rst

```diff
@@ -217,3 +217,18 @@ warning like so:
         # do stuff here
 
 .. accordion-footer::
+
+.. accordion-header::
+    :id: collapseHowToSixteen
+    :title: Display progress bar
+
+Gammapy provides the possibility of displaying a `tqdm`
```

Comment:
maybe write : displaying a progress bar...

And then "To activate this functionality, make sure `tqdm` is installed and add the following code snippet to your code:"